### PR TITLE
Adding India to region (r1)

### PR DESCRIPTION
**Pull request by raghavrithvik**
India seems to have another region code. Haven't fully tested this.

### CODE REVIEW

File: kodaksmarthome/constants.py [hunk 1]
```diff
@@ -31,7 +31,14 @@
         "URL_TOKEN": "https://api-t01-r3.perimetersafe.com/v1/oauth/token",
         "URL_AUTH": "https://app-hk.kodaksmarthome.com/web/authenticate",
         "URL_DEVICES": "https://app-hk.kodaksmarthome.com/web/user/device",
-        "URL_LOGOUT": "https://app-.kodaksmarthome.com/web/#/user/logout",
```
Comment (kairoaraujo) [inline kodaksmarthome/constants.py:34]: Fix the conflicts with master

**Review by kairoaraujo — CHANGES_REQUESTED** [review]
Fix the conflicts with master 
Also we need some tests those urls work 🤔

**Comment by raghavrithvik** [comment]
Installing HA on Python-env today. Will test before updating more. Thanks

**Comment by raghavrithvik** [comment]
Giving up. I have no clue what to do to test out the URLs. Tried a venv install with local python package and got the same 'incorrect user/pw error".

**Comment by raghavrithvik** [comment]
Changes made and tested. However camera stream not working. Entity appears though.

**Comment by raghavrithvik** [comment]
reopened pull

**Comment by raghavrithvik** [comment]
You may approve this PR. India region seems to login correctly. Might have to find a different proxy link for the camera stream. Any luck with that?

**Pull request merged**